package.json

```diff
@@ -190,6 +190,7 @@
     "schema": "node ./data/getSchema",
     "params": "node ./data/getTexts > src/params.js",
     "eslint": "./node_modules/.bin/eslint --ext js --ext jsx --ext ts --ext tsx --ignore-pattern '**/__generated__/**' --ignore-pattern '**/*.min.js' --ignore-pattern '**/setupJest.js' --ignore-pattern '**/gqlHelper.js' --ignore-pattern '**/formHooks.jsx' --ignore-pattern '**/*.stories.jsx' --fix src data",
+    "eslint-new": "./node_modules/.bin/eslint --ext js --ext jsx --ext ts --ext tsx --ignore-pattern '**/__generated__/**' --ignore-pattern '**/*.min.js' --ignore-pattern '**/setupJest.js' --ignore-pattern '**/gqlHelper.js' --ignore-pattern '**/formHooks.jsx' --ignore-pattern '**/*.stories.jsx' --fix $(git diff --name-only HEAD | xargs)",
```

Comment:
I dont think this command is behaving as expected. I tested locally and this was results after running following commands 
`npm run eslint-new`
<img width="1215" alt="Screen Shot 2022-10-24 at 10 35 38 AM" src="https://user-images.githubusercontent.com/90918464/197553043-d13ef7d7-1812-4049-8ae2-07375fc93b3d.png">
`npm run eslint`
<img width="1224" alt="Screen Shot 2022-10-24 at 10 37 09 AM" src="https://user-images.githubusercontent.com/90918464/197553111-31ada75b-f768-4933-8bb2-6a68bd78e64b.png">

Reply:
the new command for this pr picks up oddities in files we're not supposed to change and it didnt lint the random spacings I added like the original [second] command does